.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: android
+
+android:
+  components:
+    - build-tools-21.1.2
+    - android-19
+  licenses:
+    - android-sdk-license-5be876d5
+
+before_script:
+    - ./gradlew assemble
+
+script:
+    - ./gradlew check
+
+notifications:
+  email: false
```

Comment:
yeah, I had it on but felt the notification was too aggressive. 

i can re-enable this, will update.